core/help/help_scope.py

```diff
@@ -1,7 +1,8 @@
-from talon import Module, actions, imgui, scope, ui
+from talon import Context, Module, actions, imgui, scope, ui
 
+ctx = Context()
 mod = Module()
-mod.mode("help_scope", "Mode for showing the scope help gui")
+mod.tag("help_scope", "tag for showing the scope help gui")
```

Comment:
Should we named this tag `helps_scope_open` to be consistent with `help_open`?

Reply:
Make sense, done